index.html

```diff
@@ -457,39 +459,39 @@ <h3>Default Reading Order</h3>
 
 				<pre class="example" title="Audiobook Reading Order for a Single Resource">
 							{
-								"@context" : ["https://schema.org", "https://www.w3.org/ns/pub-context"],
-								"conformsTo" : "https://www.w3.org/TR/audiobooks/",
-								"url" : "https://publisher.example.org/janeeyre",
-								"name" : "Jane Eyre",
-								"readingOrder" : [{
-									"type" : "LinkedResource",
-									"url" : "audio/janeeyre.mp3",
-									"encodingFormat" : "audio/mp3",
-									"name" : "Jane Eyre",
-									"duration" : "PT124503.123S"
-								}]
+							    "@context" : ["https://schema.org", "https://www.w3.org/ns/pub-context"],
+							    "conformsTo" : "https://www.w3.org/TR/audiobooks/",
+							    "url" : "https://publisher.example.org/janeeyre",
+							    "name" : "Jane Eyre",
+							    "readingOrder" : [{
+							        "type" : "LinkedResource",
+							        "url" : "audio/janeeyre.mp3",
+							        "encodingFormat" : "audio/mp3",
+							        "name" : "Jane Eyre",
+							        "duration" : "PT124503.123S"
+							    }]
 							}
 					</pre>
 
 				<pre class="example" title="Audiobook Reading Order for Multiple Resources using Media Fragments">
 							{
-								"@context" : ["https://schema.org", "https://www.w3.org/ns/pub-context"],
-								"conformsTo" : "https://www.w3.org/TR/audiobooks/",
-								"url" : "https://publisher.example.org/janeeyre",
-								"name" : "Jane Eyre",
-								"readingOrder" : [{
-									"type": "LinkedResource",
-									"url" : "audio/part001.wav#t=0,457.931",
-									"encodingFormat" : "audio/vnd-wav",
-									"name" : "Chapter 1",
-									"duration" : "PT457.931S"
-								}, {
-									"type" : "LinkedResource",
-									"url" : "audio/part002.wav#t=12.741",
-									"encodingFormat" : "audio/vnd-wav",
-									"name" : "Chapter 2",
-									"duration" : "PT234.245S"
-								}]
+							    "@context" : ["https://schema.org", "https://www.w3.org/ns/pub-context"],
+							    "conformsTo" : "https://www.w3.org/TR/audiobooks/",
+							    "url" : "https://publisher.example.org/janeeyre",
+							    "name" : "Jane Eyre",
+							    "readingOrder" : [{
+							        "type": "LinkedResource",
+							        "url" : "audio/part001.wav#t=0,457.931",
+							        "encodingFormat" : "audio/vnd-wav",
+							        "name" : "Chapter 1",
+							        "duration" : "PT457.931S"
+							    }, {
+							        "type" : "LinkedResource",
+							        "url" : "audio/part002.wav#t=12.741",
+							        "encodingFormat" : "audio/vnd-wav",
+							        "name" : "Chapter 2",
+							        "duration" : "PT234.245S"
+						 	    }]
```

Comment:
There's a space then a tab here, so the tab is coming out in the rendered example.

(Cool, though, I never knew respec stripped leading tabs up to spaces inside pre. Learn something new every day...)